### DISCUSSION TITLE
Cruiser routing menu

### DISCUSSION THREAD

**Discussion by 0709wiwiwi** [devemux86/cruiser#193] [Q&A] [answered]
**Cruiser>Settings>Routing.** 
_What is needed less often in Settings Routing_
Contains items that usually are set once 
Like Vehicle speed & Voice distances.

**Cruiser>Routing Panel.** 
_What is often needed grouped in the Routing panel._
Contains items that are often needed.

**Comment by devemux86** [comment]
@0709wiwiwi thanks for the detailed description!

When you write "menu" you probably mean "settings"?

I prefer not to expose app files in folders where users can edit / delete them and could break the app operation.
We can use "Open X" actions to use external files. Like already happens with maps, themes, BRouter profiles.

The only files that could currently be placed in public folders are the (map) downloads, mainly due to their size.

**Comment by devemux86** [comment]
We can put the Profile + Service in the title, which is clickable and allows to change them.
Maybe not obvious, but we don't load the menu with functions that many not change often?

<img src="https://github.com/devemux86/cruiser/assets/3484020/38d6c42c-8869-43f1-ba42-ce3167c076f2" width="50%"/>

**Reply by devemux86** [reply, marked as answer]
> We can put the Profile + Service in the title, which is clickable and allows to change them.

Available in [Cruiser 3.0.23](https://play.google.com/store/apps/details?id=gr.talent.cruiser).
